react/src/components/search-app/SearchResultsComponent.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+
+export default class SearchResultsComponent extends React.Component {
```

Comment:
Now that I'm thinking about it, if we don't have any interactivity with the search results, why don't we render them on the server instead of passing this `dataForReact` object?